src/components/DocumentControlsPanel/DocumentActionsPanel/DocumentActionsPanel.cypress.js

```diff
@@ -19,18 +19,18 @@ describe("DocumentActions", function() {
     cy.visit(`?collection=${this.collectionId}`);
 
     // Click delete to delete the default selected document
-    cy.get("[data-testid=delete_button]").click();
+    cy.get("[aria-label='Delete button']").click();
```

Comment:
aria-label isn't necessary for a well labeled button.

Reply:
You can use https://testing-library.com/docs/cypress-testing-library/intro to write the same queries in Cypress tests that you are in your react-testing-library tests. That would simplify these queries.

Reply:
In this case, the delete button is a trash can icon only (no text). Therefore, it would make sense to have an aria-label, right (which we would also use for testing here, presumably)?

Reply:
Oops, sorry. Agreed. 👍